wagtail/admin/forms/tags.py

```diff
@@ -31,8 +33,18 @@ def __init__(self, *args, **kwargs):
             self.widget.free_tagging = self.free_tagging
 
     def clean(self, value):
+
         value = super().clean(value)
 
+        value_too_long = ""
+        for val in value:
+            if len(val) > 100:
+                if value_too_long:
+                    value_too_long += ", "
+                value_too_long += val
+        if value_too_long:
+            raise ValidationError(_(f"Tag(s) {value_too_long} are over 100 characters"))
```

Comment:
Is this value of `100` available anywhere, just thinking it might be good to avoid hard-coding this.

Reply:
I can't find anywhere else it might be available. Should I create a variable for it before the declaration of the class?

Something like `MAX_TAG_LENGTH = 100`

Reply:
It looks like the max length is on the tag name & slug field. 

https://github.com/jazzband/django-taggit/blob/master/taggit/models.py#L17

So maybe you can pull it from the model, something like. 

`self.tag_model.name.max_length`

Reply:
Hi, I have made the changes. Please comment if you feel it still needs to be fixed. Thank you